security/docker-compose.yml

```diff
@@ -54,8 +54,7 @@ services:
   opi-security-client:
     build:
       context: client
-    networks:
-      - internet
+    network_mode: service:vpn-server
```

Comment:
try
```
+ docker-compose up opi-security-client
```
instead of
```
+ docker-compose run opi-security-client
```
maybe this will help you with CI error... but notice the error code... UP probably will not fail if client existed, vs RUN will fail properly

Reply:
This worked, thanks!